examples/webgpu_compute.html

```diff
@@ -116,19 +116,24 @@
 
 				// compute
 
-				computeNode = compute( particleNum );
-
-				computeNode.computeNode = FnNode;
+				computeNode = compute( FnNode, particleNum );
 
 				// use a compute shader to animate the point cloud's vertex data.
 
+				const particleNode = attribute( 'particle', 'vec2' );
+
 				const pointsGeometry = new THREE.BufferGeometry();
-				pointsGeometry.setAttribute( 'position', particleBuffer );
+				pointsGeometry.setAttribute( 'position', new THREE.BufferAttribute( new Float32Array( 3 ) ) ); // single vertex ( not triangle )
+				pointsGeometry.setAttribute( 'particle', particleBuffer ); // dummy the position points as instances
+				pointsGeometry.drawRange.count = 1; // force render points as instances ( not triangle )
 
 				const pointsMaterial = new Nodes.PointsNodeMaterial();
-				pointsMaterial.colorNode = add( positionLocal, color( 0xFFFFFF ) );
+				pointsMaterial.colorNode = add( particleNode, color( 0xFFFFFF ) );
+				pointsMaterial.positionNode = particleNode;
 
 				const mesh = new THREE.Points( pointsGeometry, pointsMaterial );
+				mesh.isInstancedMesh = true;
+				mesh.count = particleNum;
```

Comment:
Hmmm... Someday we'll have to figure out how to make a better abstraction for instancing...

Reply:
I think in something like this: https://github.com/mrdoob/three.js/pull/23835#discussion_r844829963

```js
const instancedObj3d = new InstanceObject3D( mesh || sprite3d || skinnedMesh, instanceCount );
instancedObj3d.target; // mesh || sprite3d || skinnedMesh
instancedObj3d.count; // instanceCount 

scene.add( instancedObj3d  );
```